pandas/tools/util.py

```diff
@@ -173,6 +174,8 @@ def to_numeric(arg, errors='raise', downcast=None):
             values = arg.values
     elif isinstance(arg, (list, tuple)):
         values = np.array(arg, dtype='O')
+    elif isinstance(arg, Decimal):
```

Comment:
@jreback I can easily move the ``isinstance(value, Decimal)`` check to the isscalar function, but I don't see the point, I'd still have to check again after this in order to call ``float(value)``